endpoint/endpoint.go

```diff
@@ -376,16 +375,7 @@ func (e *Endpoint) Describe() string {
 
 // UniqueOrderedTargets removes duplicate targets from the Endpoint and sorts them in lexicographical order.
 func (e *Endpoint) UniqueOrderedTargets() {
```

Comment:
:thinking: Shouldn't this func be (at least) renamed ? or removed / merged with NewTargets ?